### PR TITLE
Fix/pcc operation

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: client/app/dedicatedCloud/operation/dedicatedCloud-operation.controller.js [hunk 1]
```diff
@@ -1,221 +1,163 @@
...
-        self.operations = [];
-        self.stateEnum = [];
-        self.stateFilter = null;
-        self.doneStates = ["canceled", "done"];
-        self.loaders = {
-            operations: false,
-            models: false,
-            updating: false
-        };
+angular.module("App").controller("DedicatedCloudOperationsCtrl", function ($q, $scope, $state, $stateParams, $translate, $window, BillingOrders, Alerter, DedicatedCloud, ouiDatagridService) {
```
Comment (euhmeuh) [inline client/app/dedicatedCloud/operation/dedicatedCloud-operation.controller.js]: Why not a class?

Comment (frenautvh) [reply client/app/dedicatedCloud/operation/dedicatedCloud-operation.controller.js]: why a class ? ^^

Comment (euhmeuh) [reply client/app/dedicatedCloud/operation/dedicatedCloud-operation.controller.js]: :crying_cat_face:

**Comment by cbourgois** [comment]
Some conflicts (ui-kit, ovh-api-services)

**Pull request merged**